src/pynwb/data/nwb.base.yaml

```diff
@@ -91,6 +91,24 @@ groups:
 - neurodata_type_def: NWBDataInterface
   neurodata_type_inc: NWBContainer
   doc: An abstract data type for differentiating experimental data from metadata
+- neurodata_type_def: Image
```

Comment:
Why is Image a group and not a dataset? Making a group requires the additional dataset. This doubles the number of objects need to store an image